code/socialDistribution/requests.py

```diff
@@ -117,11 +117,20 @@ def delete(url, params=None):
     """
 
     headers = {
-        "Accept": "application/json",
-        "REFERER": HOST
+        "Accept": "application/json"
     }
 
-    response = requests.delete(url, headers=headers, params=params)
+    # ref: https://stackoverflow.com/questions/15431044/can-i-set-max-retries-for-requests-request - datashaman
+    # 'Can I set max_retries for requests.request?'
+    session = requests.Session()
```

Comment:
consider wrapping this in try-except as well as these lines could fail for many reasons, similar to get above. thanks